src/components/examples/TimeDisplay.tsx

```
@@ -0,0 +1,120 @@
+
+    return () => clearInterval(interval);
+  }, [realTime, updateInterval]);
+
+  const formatTime = (date: Date): string => {
+    switch (format) {
+      case 'time':
+        return date.toLocaleTimeString();
+      case 'date':
+        return date.toLocaleDateString();
+      case 'relative':
+        const now = new Date();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)